### ISSUE TITLE
Slider is not accessible via keyboard (accessibility issue)

### ISSUE THREAD

**Issue by GenevieveTheSweet** [mendixlabs/boolean-slider#22]
Our site needs to be accessible and we would love to use the slider but it is currently not able to interact with the keyboard. There should be functionality that allows the user to tab to the slider and then press a keyboard button (usually space bar) to toggle it.

**Comment by Andries-Smit** [comment]
Thank you for reporting this issue. We are planning to improve the accessibility for all our widgets, though the planning in not clear when. we will keep you informed. Cheers, Andries
